packages/react-integration/demo-app-ts/src/components/demos/TopologyDemo/Connectors.tsx

```diff
@@ -210,9 +210,18 @@ export const CreateConnector = withTopologySetup(() => {
         }
         if (kind === ModelKind.node) {
           return withCreateConnector(
-            (source: Node, target: Node | Graph, event: DragEvent, choice: ColorChoice | undefined): any[] | null => {
+            (
+              source: Node,
+              target: Node | Graph,
+              event: DragEvent,
+              dropHints: string[],
+              choice: ColorChoice | undefined
+            ): Promise<any[] | null> => {
               if (!choice) {
-                return [{ label: 'Create Annotation', color: 'red' }, { label: 'Create Binding', color: 'green' }];
+                return Promise.resolve([
+                  { label: 'Create Annotation', color: 'red' },
+                  { label: 'Create Binding', color: 'green' }
+                ]);
```

Comment:
Remove `Promise` from this example:
```suggestion
                return [{ label: 'Create Annotation', color: 'red' }, { label: 'Create Binding', color: 'green' }];
```